nucypher/blockchain/eth/actors.py

```diff
@@ -666,20 +649,23 @@ def confirm_activity(self) -> str:
     def _confirm_period(self) -> None:
         # TODO: Follow-up actions for downtime
         # TODO: Check for stake expiration and exit
-        missed_periods = self.stake_tracker.current_period - self.last_active_period
+        missed_periods = self.period_tracker.current_period - self.last_active_period
         if missed_periods:
             self.log.warn(f"MISSED CONFIRMATIONS - {missed_periods} missed staking confirmations detected!")
         self.confirm_activity()  # < --- blockchain WRITE
-        self.log.info("Confirmed activity for period {}".format(self.stake_tracker.current_period))
+        self.log.info("Confirmed activity for period {}".format(self.period_tracker.current_period))
```

Comment:
Recall that activity is confirmed for the *next period* (yeah, crazy, ... #1272).
Suggested change:
```python
if self.period_tracker.current_period >= self.last_active_period:
    missed_periods = self.period_tracker.current_period - self.last_active_period + 1
    self.log.warn(f"MISSED CONFIRMATIONS - {missed_periods} missed staking confirmations detected!")
    self.confirm_activity()  # < --- blockchain WRITE
    self.log.info("Confirmed activity for period {}".format(self.period_tracker.current_period + 1))
```